tools/nni_cmd/nnictl.py

```diff
@@ -129,6 +130,16 @@ def parse_args():
     parser_experiment_clean.add_argument('id', nargs='?', help='the id of experiment')
     parser_experiment_clean.add_argument('--all', action='store_true', default=False, help='delete all of experiments')
     parser_experiment_clean.set_defaults(func=experiment_clean)
+    parser_experiment_head = parser_experiment_subparsers.add_parser('head', help='list the id and results of the' \
+                                                                                  'n trials with the highest metric')
+    parser_experiment_head.add_argument('id', nargs='?', help='the id of experiment')
+    parser_experiment_head.add_argument('--num', default=10, help='the number of items to be listed')
+    parser_experiment_head.set_defaults(func=experiment_head)
```

Comment:
Can `nnictl trial ls`  with sorting  do the same work?

Reply:
Users usually want to focus on top maximal/minimal trials. I think `nnictl trials ls` offer too much info and users cannot clearly see the overall situation of the experiment.
I want to provide an interface to the general situation of the experiment just like ↓.
![image](https://user-images.githubusercontent.com/14017336/88995552-88bfb980-d31d-11ea-9ac4-6286a0c0d3ea.png)

Reply:
OK, got it,  suggest to reuse `nnictl trial`  command since we are listing the trials. 
Consider  following options :
1.  add options to `nnictl trial ls`,  such as `nnictl trial ls -n 10 -sort asc`
2.  `nnictl trial head 10`   , `nnictl trial tail 10`

Reply:
The latter looks good to me. How about `nnictl trial head 10 --reverse True` instead of `nnictl trial tail 10`.

Reply:
`head 10 --reverse`   maybe misunderstood as  reverse sorting the the top 10,   not  bottom 10.   The meaning of `tail 10` is clear.